### DISCUSSION TITLE
Selective Install Vanna

### DISCUSSION THREAD

**Discussion by wienzzz** [vanna-ai/vanna#345] [Q&A]
Hello, I am leaning toward Vanna for my chatbot/copilot use case and I plan to make this serverless.

When I install plain vanna using `pip install vanna` this will generate a whooping 420MB libraries

Is there any specific information (or which packages to install) if I only want to use vanna in order to communicate  with LLM and vector store only without any need to actually execute the SQL on SQL database and or perform any visualization?

**Comment by zainhoda** [comment]
Unfortunately this is the minimal installation that will work, which is 330MB

```bash
pip install vanna --no-dependencies
pip install tabulate
pip install pandas
pip install requests
pip install plotly
```
